stock_operating_unit/model/stock.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp.tools.translate import _
+from openerp import api, fields, models
```

Comment:
`_` can be imported from the same package: `from openerp import _, api, fields, models`